reference-architecture/azure-ansible/bastion.sh

```diff
@@ -0,0 +1,345 @@
+  - name: install the latest version of PyYAML
+    yum: name=PyYAML state=latest
+  - name: Install the ose client
+    yum: name=atomic-openshift-clients state=latest
+  - name: Update all hosts
+    command: yum -y update
+    async: 1200
+    poll: 10
+  - name: Wait for Things to Settle
+    pause:  minutes=5
+EOF
+
+cat <<EOF > /home/${AUSERNAME}/quota.yml
+---
+- hosts: quotanodes
+  vars:
+    description: "Fix EP Storage/Quota"
+  tasks:
+  - name: Format xfs on disk
+    filesystem: dev=/dev/sdd fstype=xfs force=yes
+  - name: Update Mount to Handle Quota
+    mount: fstype=xfs name=/var/lib/origin/openshift.local/volumes src=/dev/sdd opts="gquota" state="mounted"
+  - name: Change Node perFSGroup Qouta Setting
+    lineinfile: dest=/etc/origin/node/node-config.yaml regexp=^perFSGroup line="    perFSGroup:512Mi"
+EOF
+
+
+cat <<EOF > /home/${AUSERNAME}/setupiscsi.yml
+- hosts: all
+  vars:
+    description: "Subscribe OSE"
+  tasks:
+  - name: Install iscsi initiator utils
+    yum: name=iscsi-initiator-utils state=latest
+  - name: add new initiator name
+    lineinfile: dest=/etc/iscsi/initiatorname.iscsi create=yes regexp="InitiatorName=*" line="InitiatorName=iqn.2016-02.local.azure.nodes" state=present
+  - name: restart iscsid service
+    shell: systemctl restart iscsi
+    ignore_errors: yes
+  - name: Enable Iscsi
+    shell: systemctl enable iscsi
+    ignore_errors: yes
+  - name: Start iScsi Initiator  Service
+    shell: systemctl start iscsi
+    ignore_errors: yes
+  - name: Discover Devices on Iscsi  All Hosts
+    shell: iscsiadm --mode discovery --type sendtargets --portal store1
+    register: task_result
+    until: task_result.rc == 0
+    retries: 10
+    delay: 30
+    ignore_errors: yes
+  - name: Login All Hosts
+    shell: iscsiadm --mode node --portal store1 --login
+    register: task_result
+    until: task_result.rc == 0
+    retries: 10
+    delay: 30
+    ignore_errors: yes
+EOF
+
+
+cat <<EOF > /home/${AUSERNAME}/postinstall.yml
+---
+- hosts: masters
+  vars:
+    description: "auth users"
+  tasks:
+  - name: Create Master Directory
+    file: path=/etc/origin/master state=directory
+  - name: add initial user to OpenShift Enterprise
+    shell: htpasswd -c -b /etc/origin/master/htpasswd ${AUSERNAME} ${PASSWORD}
+
+EOF
+
+cat <<EOF > /home/${AUSERNAME}/openshift-install.sh
+export ANSIBLE_HOST_KEY_CHECKING=False
+sleep 120
+ansible all --module-name=ping > ansible-preinstall-ping.out || true
+ansible-playbook  /home/${AUSERNAME}/subscribe.yml
+echo "${RESOURCEGROUP} Bastion Host is starting ansible BYO" | mail -s "${RESOURCEGROUP} Bastion BYO Install" ${RHNUSERNAME} || true
+ansible-playbook  /usr/share/ansible/openshift-ansible/playbooks/byo/config.yml < /dev/null &> byo.out
+
+# ssh gwest@master1 oadm registry --selector=region=infra
+# ssh gwest@master1 oadm router --selector=region=infra
+wget http://master1:8443/api > healtcheck.out
```

Comment:
I prefer wget also, the only reason I brought this up was because you mentioned that your installs were taking 4 hours, so I was looking for places to speed up your install. The "yum -y install wget" on line 44 can sometimes take upwards of 3-4 minutes when mirrors are slow. You used curl in the azure-ansible/store.sh script so I just assumed you were comfortable with both. Feel free to call this comment resolved.